modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectDefinitionLocalServiceTest.java

```diff
@@ -3106,6 +3278,20 @@ private void _assertModelResourceNames(List<String> objectDefinitionNames)
 		}
 	}
 
+	private void _assertObjectDefinitionSetting(
```

Comment:
Rename to _**assertObjectDefinitionSettingsValues**

Reply:
Done